packages/ui-extensions/src/surfaces/admin/extension-targets.ts

```diff
@@ -49,11 +54,11 @@ export interface ExtensionTargets {
   Playground: RenderExtension<StandardApi<'Playground'>, AllComponents>;
 
   /**
-   * @private
+   * Renders a [CustomerSegmentTemplate](/docs/api/admin-extensions/components/customersegmenttemplate) in the customer segment editor.
```

Comment:
what is the customer segment editor? consider a link to merchant-facing docs about this editor